protos/local/workflow_activation.proto

```diff
@@ -59,9 +59,8 @@ message WFActivationJob {
         /// after performing the activation.
         ///
         /// If other job variant are present in the list, this variant will be the last job in the
-        /// job list. The boolean value is irrelevant, since the variant type is what matters. It
-        /// will be set to true if this is the variant.
-        bool remove_from_cache = 50;
+        /// job list. The string value is a reason for eviction.
+        RemoveFromCache remove_from_cache = 50;
```

Comment:
I trust we can guarantee that these protos are only used for FFI and never stored or used across versions and therefore are not concerned with compatibility? Or at least while we're in beta we can assume that the typescript SDK version always matches the exact expected core version?

Reply:
Yes exactly